ste/sender-blobFolders.go

```diff
@@ -214,9 +214,10 @@ func (b *blobFolderSender) SetFolderProperties() error {
 }
 
 func (b *blobFolderSender) DirUrlToString() string {
-	url := b.destination.URL()
-	url.RawQuery = ""
-	return url.String()
+	uri, _ := url.Parse(b.jptm.Info().Destination)
+	uri.RawPath = ""
```

Comment:
Why strip off the path name here?

Reply:
Keep in mind that RawPath is only to specify whether characters are encoded; namely for characters that don't need to be encoded.

This just normalizes the path.